### DISCUSSION TITLE
Hidden Files

### DISCUSSION THREAD

**Discussion by pablofvila** [DirectoryLister/DirectoryLister#749] [Help & Support] [answered]
Hi!, first of all, Excellent work, love this!
But i had a problem hidden files, don't hide, but if i put an * works hidden all.
What i doing wrong?

my .env has:
HIDDEN_FILES_LIST=.hidden

i created in same dir of index.php the file .hidden to with content:

'hidden_files' => [
    'favicon.ico',
    'OLD',
]

but still seeing the files.

Hope you can help.

Thanks a lot!

**Comment by Bubba8291** [comment, marked as answer]
I was just about to create an issue about hidden files. The `.hidden` file doesn't need a php array. It only needs the filenames you want to hide line by line. Everything you need to know is in the [docs](https://docs.directorylister.com/configuration/hiding-files).

About my issue, I can't seem to hide files that are inside of a symbolic link. I tried a bunch of different formats such as putting the `.hidden` file in the symbolic link directory. I also tried using the full path to the directory of the symbolic link in the `.hidden` file. None of those worked. Also keep in mind that I modified the `open_basedir` to allow access to the directory of the symbolic link. This isn't an issue with the `.hidden` file itself as I have a directory in the web root that isn't a symbolic and is hidden. Anyone know how I could go about this?

**Reply by pablofvila** [reply]
Great!, now works like a charm.
I don't have symbolic links, but i can test it later.
Thanks a lot!

**Reply by githubsean** [reply]
I'm getting the issue with files not being hidden if in symbolic links. And as @Bubba8291 has done, I have set open_basedir appropriately.
My use case is I have all my files symbolically linked:

Movies -> /mnt/nas-media/Video/Movies
TV -> /mnt/nas-media/Video/TV
